pkg/config/linters_settings.go

```diff
@@ -179,6 +179,12 @@ var defaultLintersSettings = LintersSettings{
 		HTTPMethod:     true,
 		HTTPStatusCode: true,
 	},
+	UseTesting: UseTestingSettings{
+		ContextBackground: true,
+		ContextTodo:       true,
+		OSChdir:           true,
+		OSMkdirTemp:       true,
+	},
```

Comment:
Let's set explicitly:
```suggestion
		OSSetenv:          false,
	},
```